README.md

```diff
@@ -166,15 +169,19 @@ bynder
 
 ### Upload
 
+- `prepareUpload()`
+- `uploadFileInChunks(fileObject, fileId, fileSize)`
+- `saveAsset(queryObject)`
 - `uploadFile(fileObject)`
+- `finaliseUpload(fileObject, fileName, chunks, fileSize)`
```

Comment:
Wouldn't there still be just one `uploadFile(fileObject)` for the user? Wouldn't make sense to me to expose the internal calls like prepare/uploadChunk/etc. as well here.

Reply:
I agree. Although the old methods have JS docs linking to our APIary docs.

However, I'll make those methods private now.